src/textual/app.py

```diff
@@ -87,7 +98,9 @@ class ActionError(Exception):
 class App(Generic[ReturnType], DOMNode):
     """The base class for Textual Applications"""
 
-    css = ""
+    CSS = """
```

Comment:
Just to be sure I understand correctly... The only reason we can move this to a class-level variable that can no longer be overridden in the constructor is because we expect to never have more than one instance of App instantiated in a Python program, right? :slightly_smiling_face: 
(so customising this CSS at the class level can only be be done before instantiating an App for the first time)

Reply:
You can still define a subclass with different CSS.

I wanted to remove it from the constructor so that there is only one place to included the CSS if its a string literal. A classvar constant seems like the best place.